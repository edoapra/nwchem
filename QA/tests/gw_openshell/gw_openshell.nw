title "GW Trifluoroacetamide"
start
echo

memory 2000 mb

geometry
O   1.5154     1.1380    -0.0012
N   1.4926    -1.1655    -0.0010
C   0.9005     0.0760     0.0021
H   2.5025    -1.2628    -0.0022
H   0.9511    -2.0241    -0.0008
end

basis "ao basis" bse spherical
 * library cc-pvdz
end

basis "cd basis" bse spherical
 * library cc-pvdz-rifit
end


dft
 odft
 mult 2
 xc pbe0
 direct
 grid fine nodisk
 tolerances accCoul 12
 convergence energy 1d-8
 noprint "final vectors analysis"
end

gw
 method cdgw
 states alpha occ 5 vir 5
 states beta occ 5 vir 5
end

task dft gw

