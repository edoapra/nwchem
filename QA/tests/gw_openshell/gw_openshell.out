 argument  1 = /people/meji656/calculations/bse/QA/gw_openshell.nw
  NWChem w/ OpenMP: maximum threads =    1



============================== echo of input deck ==============================
title "GW Trifluoroacetamide"
start
echo

memory 2000 mb

geometry
O   1.5154     1.1380    -0.0012
N   1.4926    -1.1655    -0.0010
C   0.9005     0.0760     0.0021
H   2.5025    -1.2628    -0.0022
H   0.9511    -2.0241    -0.0008
end

basis "ao basis" bse spherical
 * library cc-pvdz
end

basis "cd basis" bse spherical
 * library cc-pvdz-rifit
end


dft
 odft
 mult 2
 xc pbe0
 direct
 grid fine nodisk
 tolerances accCoul 12
 convergence energy 1d-8
 noprint "final vectors analysis"
end

gw
 method cdgw
 states alpha occ 5 vir 5
 states beta occ 5 vir 5
end

task dft gw

================================================================================


                                         
                                         


             Northwest Computational Chemistry Package (NWChem) 7.3.0
             --------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2025
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = dc006
    program         = /qfs/people/meji656/software/nwchem.bse/bin/LINUX64/nwchem
    date            = Thu Oct 16 10:35:25 2025

    compiled        = Thu_Oct_16_10:07:15_2025
    source          = /qfs/people/meji656/software/nwchem.bse
    nwchem branch   = 7.3.0
    nwchem revision = v7.2.0-beta1-1624-g0dc011d0a0
    ga revision     = 5.9.2
    use scalapack   = F
    input           = /people/meji656/calculations/bse/QA/gw_openshell.nw
    prefix          = gw_openshell.
    data base       = ./gw_openshell.db
    status          = startup
    nproc           =        7
    time left       =     -1s



           Memory information
           ------------------

    heap     =     65535998 doubles =      500.0 Mbytes
    stack    =     65535995 doubles =      500.0 Mbytes
    global   =    131072000 doubles =     1000.0 Mbytes (distinct from heap & stack)
    total    =    262143993 doubles =     2000.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------


                               GW Trifluoroacetamide
                               ---------------------

 Scaling coordinates for geometry "geometry" by  1.889725989
 (inverse scale =  0.529177249)

 C1  symmetry detected

          ------
          auto-z
          ------
  no constraints, skipping    0.0000000000000000     
  no constraints, skipping    0.0000000000000000     


                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in angstroms (scale by  1.889725989 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 O                    8.0000     0.16028417     1.21853876    -0.00027246
    2 N                    7.0000     0.11610501    -1.08465041    -0.00020670
    3 C                    6.0000    -0.46445442     0.16229164     0.00052807
    4 H                    1.0000     1.12505507    -1.19131944     0.00266455
    5 H                    1.0000    -0.43333700    -1.93818759    -0.00220638

      Atomic Mass 
      ----------- 

      O                 15.994910
      N                 14.003070
      C                 12.000000
      H                  1.007825


 Effective nuclear repulsion energy (a.u.)      63.3083784987

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000



                                Z-matrix (autoz)
                                -------- 

 Units are Angstrom for bonds and degrees for angles

      Type          Name      I     J     K     L     M      Value
      ----------- --------  ----- ----- ----- ----- ----- ----------
    1 Stretch                  1     3                       1.22717
    2 Stretch                  2     3                       1.37547
    3 Stretch                  2     4                       1.01458
    4 Stretch                  2     5                       1.01509
    5 Bend                     1     3     2               124.43086
    6 Bend                     3     2     4               121.00086
    7 Bend                     3     2     5               122.26360
    8 Bend                     4     2     5               116.73547
    9 Torsion                  1     3     2     4           0.27386
   10 Torsion                  1     3     2     5        -179.81955


            XYZ format geometry
            -------------------
     5
 geometry
 O                     0.16028417     1.21853876    -0.00027246
 N                     0.11610501    -1.08465041    -0.00020670
 C                    -0.46445442     0.16229164     0.00052807
 H                     1.12505507    -1.19131944     0.00266455
 H                    -0.43333700    -1.93818759    -0.00220638

 ==============================================================================
                                internuclear distances
 ------------------------------------------------------------------------------
       center one      |      center two      | atomic units |  angstroms
 ------------------------------------------------------------------------------
    3 C                |   1 O                |     2.31902  |     1.22717
    3 C                |   2 N                |     2.59926  |     1.37547
    4 H                |   2 N                |     1.91727  |     1.01458
    5 H                |   2 N                |     1.91825  |     1.01509
 ------------------------------------------------------------------------------
                         number of included internuclear distances:          4
 ==============================================================================



 ==============================================================================
                                 internuclear angles
 ------------------------------------------------------------------------------
        center 1       |       center 2       |       center 3       |  degrees
 ------------------------------------------------------------------------------
    3 C                |   2 N                |   4 H                |   121.00
    3 C                |   2 N                |   5 H                |   122.26
    4 H                |   2 N                |   5 H                |   116.74
    1 O                |   3 C                |   2 N                |   124.43
 ------------------------------------------------------------------------------
                            number of included internuclear angles:          4
 ==============================================================================





 Summary of "ao basis" -> "" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 *                          cc-pvdz                   on all atoms 




 Summary of "cd basis" -> "" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 *                       cc-pvdz-rifit                on all atoms 



                                 NWChem GW Module
                                 ----------------


                               GW Trifluoroacetamide



                                 NWChem DFT Module
                                 -----------------


                               GW Trifluoroacetamide


                      Basis "ao basis" -> "ao basis" (spherical)
                      -----
  O (Oxygen)
  ----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  1.17200000E+04  0.000710
  1 S  1.75900000E+03  0.005470
  1 S  4.00800000E+02  0.027837
  1 S  1.13700000E+02  0.104800
  1 S  3.70300000E+01  0.283062
  1 S  1.32700000E+01  0.448719
  1 S  5.02500000E+00  0.270952
  1 S  1.01300000E+00  0.015458

  2 S  1.17200000E+04 -0.000160
  2 S  1.75900000E+03 -0.001263
  2 S  4.00800000E+02 -0.006267
  2 S  1.13700000E+02 -0.025716
  2 S  3.70300000E+01 -0.070924
  2 S  1.32700000E+01 -0.165411
  2 S  5.02500000E+00 -0.116955
  2 S  1.01300000E+00  0.557368

  3 S  3.02300000E-01  1.000000

  4 P  1.77000000E+01  0.043018
  4 P  3.85400000E+00  0.228913
  4 P  1.04600000E+00  0.508728

  5 P  2.75300000E-01  1.000000

  6 D  1.18500000E+00  1.000000

  N (Nitrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  9.04600000E+03  0.000700
  1 S  1.35700000E+03  0.005389
  1 S  3.09300000E+02  0.027406
  1 S  8.77300000E+01  0.103207
  1 S  2.85600000E+01  0.278723
  1 S  1.02100000E+01  0.448540
  1 S  3.83800000E+00  0.278238
  1 S  7.46600000E-01  0.015440

  2 S  9.04600000E+03 -0.000153
  2 S  1.35700000E+03 -0.001208
  2 S  3.09300000E+02 -0.005992
  2 S  8.77300000E+01 -0.024544
  2 S  2.85600000E+01 -0.067459
  2 S  1.02100000E+01 -0.158078
  2 S  3.83800000E+00 -0.121831
  2 S  7.46600000E-01  0.549003

  3 S  2.24800000E-01  1.000000

  4 P  1.35500000E+01  0.039919
  4 P  2.91700000E+00  0.217169
  4 P  7.97300000E-01  0.510319

  5 P  2.18500000E-01  1.000000

  6 D  8.17000000E-01  1.000000

  C (Carbon)
  ----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  6.66500000E+03  0.000692
  1 S  1.00000000E+03  0.005329
  1 S  2.28000000E+02  0.027077
  1 S  6.47100000E+01  0.101718
  1 S  2.10600000E+01  0.274740
  1 S  7.49500000E+00  0.448564
  1 S  2.79700000E+00  0.285074
  1 S  5.21500000E-01  0.015204

  2 S  6.66500000E+03 -0.000146
  2 S  1.00000000E+03 -0.001154
  2 S  2.28000000E+02 -0.005725
  2 S  6.47100000E+01 -0.023312
  2 S  2.10600000E+01 -0.063955
  2 S  7.49500000E+00 -0.149981
  2 S  2.79700000E+00 -0.127262
  2 S  5.21500000E-01  0.544529

  3 S  1.59600000E-01  1.000000

  4 P  9.43900000E+00  0.038109
  4 P  2.00200000E+00  0.209480
  4 P  5.45600000E-01  0.508557

  5 P  1.51700000E-01  1.000000

  6 D  5.50000000E-01  1.000000

  H (Hydrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  1.30100000E+01  0.019685
  1 S  1.96200000E+00  0.137977
  1 S  4.44600000E-01  0.478148

  2 S  1.22000000E-01  1.000000

  3 P  7.27000000E-01  1.000000



 Summary of "ao basis" -> "ao basis" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 O                          cc-pvdz                  6       14   3s2p1d
 N                          cc-pvdz                  6       14   3s2p1d
 C                          cc-pvdz                  6       14   3s2p1d
 H                          cc-pvdz                  3        5   2s1p


                      Basis "cd basis" -> "cd basis" (spherical)
                      -----
  O (Oxygen)
  ----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  1.09461000E+02  1.000000

  2 S  2.55780000E+01  1.000000

  3 S  9.55148000E+00  1.000000

  4 S  2.93960000E+00  1.000000

  5 S  1.39638000E+00  1.000000

  6 S  9.05061000E-01  1.000000

  7 S  4.21376000E-01  1.000000

  8 P  3.61292000E+01  1.000000

  9 P  1.08817000E+01  1.000000

 10 P  3.20594000E+00  1.000000

 11 P  1.40387000E+00  1.000000

 12 P  6.12763000E-01  1.000000

 13 D  1.52511000E+01  1.000000

 14 D  4.52087000E+00  1.000000

 15 D  1.81221000E+00  1.000000

 16 D  5.92201000E-01  1.000000

 17 F  4.22194000E+00  1.000000

 18 F  1.79021000E+00  1.000000

  N (Nitrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  8.76864000E+01  1.000000

  2 S  2.04679000E+01  1.000000

  3 S  7.62436000E+00  1.000000

  4 S  2.34426000E+00  1.000000

  5 S  1.12324000E+00  1.000000

  6 S  7.23539000E-01  1.000000

  7 S  3.27261000E-01  1.000000

  8 P  2.92373000E+01  1.000000

  9 P  8.55299000E+00  1.000000

 10 P  2.53364000E+00  1.000000

 11 P  1.09461000E+00  1.000000

 12 P  4.61104000E-01  1.000000

 13 D  1.23278000E+01  1.000000

 14 D  3.42816000E+00  1.000000

 15 D  1.28537000E+00  1.000000

 16 D  4.77850000E-01  1.000000

 17 F  3.40118000E+00  1.000000

 18 F  1.33473000E+00  1.000000

  C (Carbon)
  ----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  6.27453000E+01  1.000000

  2 S  1.38045000E+01  1.000000

  3 S  4.77277000E+00  1.000000

  4 S  1.51333000E+00  1.000000

  5 S  7.82697000E-01  1.000000

  6 S  4.09072000E-01  1.000000

  7 S  2.06796000E-01  1.000000

  8 P  2.08851000E+01  1.000000

  9 P  6.00523000E+00  1.000000

 10 P  1.72060000E+00  1.000000

 11 P  7.54465000E-01  1.000000

 12 P  3.21667000E-01  1.000000

 13 D  1.01715000E+01  1.000000

 14 D  2.67844000E+00  1.000000

 15 D  9.22515000E-01  1.000000

 16 D  3.40842000E-01  1.000000

 17 F  2.74293000E+00  1.000000

 18 F  8.95756000E-01  1.000000

  H (Hydrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  5.11588952E+00  1.000000

  2 S  1.14296528E+00  1.000000

  3 S  2.91661530E-01  1.000000

  4 P  1.91496403E+00  1.000000

  5 P  9.86401240E-01  1.000000

  6 D  1.17208347E+00  1.000000



 Summary of "cd basis" -> "cd basis" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 O                       cc-pvdz-rifit              18       56   7s5p4d2f
 N                       cc-pvdz-rifit              18       56   7s5p4d2f
 C                       cc-pvdz-rifit              18       56   7s5p4d2f
 H                       cc-pvdz-rifit               6       14   3s2p1d




 Summary of "ao basis" -> "ao basis" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 O                          cc-pvdz                  6       14   3s2p1d
 N                          cc-pvdz                  6       14   3s2p1d
 C                          cc-pvdz                  6       14   3s2p1d
 H                          cc-pvdz                  3        5   2s1p


  Caching 1-el integrals 
  tol_rho modified to match energy
  convergence criterion.

            General Information
            -------------------
          SCF calculation type: DFT
          Wavefunction type:  spin polarized.
          No. of atoms     :     5
          No. of electrons :    23
           Alpha electrons :    12
            Beta electrons :    11
          Charge           :     0
          Spin multiplicity:     2
          Use of symmetry is: off; symmetry adaption is: off
          Maximum number of iterations:  50
          This is a Direct SCF calculation.
          AO basis - number of functions:    52
                     number of shells:    24
          A Charge density fitting basis will be used.
          CD basis - number of functions:   196
                     number of shells:    66
          Convergence on energy requested:  1.00D-08
          Convergence on density requested:  1.00D-05
          Convergence on gradient requested:  5.00D-04

              XC Information
              --------------
                         PBE0 Method XC Functional
                     Hartree-Fock (Exact) Exchange  0.250          
          PerdewBurkeErnzerhof Exchange Functional  0.750          
            Perdew 1991 LDA Correlation Functional  1.000 local    
           PerdewBurkeErnz. Correlation Functional  1.000 non-local

             Grid Information
             ----------------
          Grid used for XC integration:  fine      
          Radial quadrature: Mura-Knowles        
          Angular quadrature: Lebedev. 
          Tag              B.-S. Rad. Rad. Pts. Rad. Cut. Ang. Pts.
          ---              ---------- --------- --------- ---------
          O                   0.60       70          10.0       590
          N                   0.65       70           7.0       590
          C                   0.70       70           8.0       590
          H                   0.35       60           8.0       590
          Grid pruning is: on 
          Number of quadrature shells:   330
          Spatial weights used:  Erf1

          Convergence Information
          -----------------------
          Convergence aids based upon iterative change in 
          total energy or number of iterations. 
          Levelshifting, if invoked, occurs when the 
          HOMO/LUMO gap drops below (HL_TOL):  1.00D-02
          DIIS, if invoked, will attempt to extrapolate 
          using up to (NFOCK): 10 stored Fock matrices.

                    Damping( 0%)  Levelshifting(0.5)       DIIS
                  --------------- ------------------- ---------------
          dE  on:    start            ASAP                start   
          dE off:    2 iters         50 iters            50 iters 


      Screening Tolerance Information
      -------------------------------
          Density screening/tol_rho:  1.00D-11
          AO Gaussian exp screening on grid/accAOfunc:  18
          CD Gaussian exp screening on grid/accCDfunc:  20
          XC Gaussian exp screening on grid/accXCfunc:  20
          Schwarz screening/accCoul:  1.00D-12


      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:        -167.72839836

      Non-variational initial energy
      ------------------------------

 Total energy =    -168.813712
 1-e energy   =    -353.530316
 2-e energy   =     121.408226
 HOMO         =      -0.154097
 LUMO         =       0.070838

   Time after variat. SCF:      0.5

      3 Center 2 Electron Integral Information
      ----------------------------------------
          Maximum number of 3-center 2e- integrals is:          529984.
            This is reduced with Schwarz screening to:          367500.
            Incore requires a per proc buffer size of:           57037.
          Minimum dble words available (all nodes) is:       131065476
                   This is reduced (for later use) to:       130826784
                      proc 0 Suggested buffer size is:           57037
                         Max Suggested buffer size is:           57037
                              no. integral batches is:            1000

          Incore memory use for 3-center 2e- integrals is turned off. 
   Time prior to 1st pass:      0.5

           Memory utilization after 1st SCF pass: 
           Heap Space remaining (MW):       65.53            65530110
          Stack Space remaining (MW):       65.54            65535316

   convergence    iter        energy       DeltaE   RMS-Dens  Diis-err    time
 ---------------- ----- ----------------- --------- --------- ---------  ------
 d= 0,ls=0.0,diis     1   -168.9657539772 -2.32D+02  1.39D-02  1.93D-01     0.6
                                                     1.31D-02  1.92D-01
 d= 0,ls=0.0,diis     2   -168.6178260793  3.48D-01  1.00D-02  1.13D+00     0.8
                                                     9.06D-03  1.06D+00
 d= 0,ls=0.0,diis     3   -169.0613313228 -4.44D-01  1.36D-03  7.44D-03     0.9
                                                     1.32D-03  8.46D-03
 d= 0,ls=0.0,diis     4   -169.0630054773 -1.67D-03  9.29D-04  4.19D-03     1.1
                                                     5.76D-04  3.46D-03
 d= 0,ls=0.0,diis     5   -169.0646959497 -1.69D-03  1.55D-04  1.38D-04     1.2
                                                     1.31D-04  1.63D-04
  Resetting Diis
 d= 0,ls=0.0,diis     6   -169.0647672024 -7.13D-05  4.96D-05  7.61D-06     1.4
                                                     5.91D-05  7.90D-06
 d= 0,ls=0.0,diis     7   -169.0647730165 -5.81D-06  5.03D-05  2.16D-06     1.6
                                                     1.81D-05  8.25D-07
 d= 0,ls=0.0,diis     8   -169.0647729684  4.81D-08  1.78D-05  2.36D-06     1.7
                                                     2.14D-05  3.64D-06
 d= 0,ls=0.0,diis     9   -169.0647741451 -1.18D-06  9.22D-06  5.57D-07     1.9
                                                     4.55D-06  2.30D-07
 d= 0,ls=0.0,diis    10   -169.0647743526 -2.08D-07  9.00D-06  4.08D-08     2.0
                                                     8.19D-06  2.42D-08
 d= 0,ls=0.0,diis    11   -169.0647744288 -7.62D-08  2.13D-06  7.37D-09     2.2
                                                     1.35D-06  1.44D-09
 d= 0,ls=0.0,diis    12   -169.0647744339 -5.12D-09  7.95D-07  7.61D-10     2.4
                                                     6.20D-07  8.83D-10


         Total DFT energy =     -169.064774433940
      One electron energy =     -355.358774131041
           Coulomb energy =      144.394070494508
    Exchange-Corr. energy =      -21.408449296065
 Nuclear repulsion energy =       63.308378498658

 Numeric. integr. density =       22.999999950860

     Total iterative time =      1.9s



   alpha - beta orbital overlaps 
   ----------------------------- 


   alpha      1      2      3      4      5      6      7      8      9     10
    beta      1      2      3      4      5      6      7      8      9     10
 overlap   1.000  1.000  1.000  1.000  0.999  0.993  0.991  0.974  1.000  0.968


   alpha     11     12     13     14     15     16     17     18     19     20
    beta     11     12     13     14     15     16     17     18     19     20
 overlap   1.000  0.985  0.999  0.987  0.993  0.998  0.997  0.946  1.000  0.920


   alpha     21     22     23     24     25     26     27     28     29     30
    beta     21     22     23     24     25     26     27     28     30     29
 overlap   0.928  0.997  0.998  1.000  0.997  0.999  0.996  0.993  0.996  0.977


   alpha     31     32     33     34     35     36     37     38     39     40
    beta     31     32     33     34     35     36     37     38     39     40
 overlap   0.983  0.995  0.992  0.993  0.999  1.000  0.986  0.979  0.993  1.000


   alpha     41     42     43     44     45     46     47     48     49     50
    beta     41     42     43     44     45     46     47     48     49     50
 overlap   0.999  0.999  0.992  0.994  0.998  0.997  1.000  0.998  0.999  1.000


   alpha     51     52
    beta     51     52
 overlap   0.999  0.999

     --------------------------
     Expectation value of S2:  
     --------------------------
      <S2> =      0.7537 (Exact =     0.7500)


 center of mass
 --------------
 x =  -0.02948626 y =   0.13290476 z =  -0.00001949

 moments of inertia (a.u.)
 ------------------
         162.621484064576           0.000000000000          -0.000000000000
           0.000000000000          16.578708728791          -0.000000000000
          -0.000000000000          -0.000000000000         179.200070028499

     Multipole analysis of the density
     ---------------------------------

     L   x y z        total         alpha         beta         nuclear
     -   - - -        -----         -----         ----         -------
     0   0 0 0     -0.000000    -12.000000    -11.000000     23.000000

     1   1 0 0      0.206401      0.547712     -0.341311      0.000000
     1   0 1 0     -1.390130     -1.013204     -0.376927      0.000000
     1   0 0 1      0.000816      0.000001      0.000815      0.000000

     2   2 0 0    -11.728128    -12.959803     -9.651957     10.883633
     2   1 1 0     -0.304140      0.447969      0.218176     -0.970286
     2   1 0 1      0.006364      0.000093     -0.000746      0.007017
     2   0 2 0    -11.396501    -52.445939    -49.826305     90.875744
     2   0 1 1      0.001878      0.000424     -0.000437      0.001891
     2   0 0 2    -13.115703     -6.888717     -6.227038      0.000052

  
                             ***********************
                             ***        GW       ***
                             ***********************
  

                 Memory Information
                 ------------------
          Available GA per process is          1000.0 MBs
          Available stack per process is        500.0 MBs
          Available heap per process is         500.0 MBs

          Peak memory in Initialization
              Peak GA per process is                  0.4 MBs
              Peak stack per process is               0.0 MBs
              Peak heap per process is                0.7 MBs
              Memory not in MA per process            0.0 MBs

          Peak memory in CDGW Calculation
              Peak GA per process is                  0.5 MBs
              Peak stack per process is               0.6 MBs
              Peak heap per process is                0.7 MBs
              Memory not in MA per process            0.0 MBs

          Computing 2-center integrals
          Computing Inverse Cholesky factor
          Computing 3-center integrals
   Transform ERIs
   Distribute ERIs
   Read inverse
   Orthogonalize ERIs
          Computing Vxc
          Computing Sigma_x
          Computing W(iw) on the imaginary grid
          Solving quasiparticle equations

                  Alpha Orbitals
          State     Energy (eV)     Error (eV)
          -----------------------------------
@GW         8        -14.686           0.011
@GW         9        -13.641           0.004
@GW        10        -12.639           0.005
@GW        11        -10.354           0.002
@GW        12         -8.452           0.002
@GW        13          3.960           0.003
@GW        14          3.965           0.005
@GW        15          5.443           0.002
@GW        16         10.121           0.004
@GW        17         12.171           0.004

                   Beta Orbitals
          State     Energy (eV)     Error (eV)
          -----------------------------------
@GW         7        -17.771           0.002
@GW         8        -14.839           0.005
@GW         9        -13.398           0.002
@GW        10        -11.673           0.002
@GW        11        -10.239           0.002
@GW        12          1.142           0.002
@GW        13          4.403           0.002
@GW        14          4.180           0.005
@GW        15          5.570           0.002
@GW        16         10.106           0.003

     GW Timing Statistics
   ------------------------
   Init.   :          0.1s
     ERIs  :          0.0s
     Distr.:          0.0s
     Vxc   :          0.1s
   Vmo     :          0.0s
   iW      :          0.2s
   I_n     :          0.0s
   R_n     :          0.1s
   GW total:          0.5s


 Task  times  cpu:        2.6s     wall:        2.6s


                                NWChem Input Module
                                -------------------


 Summary of allocated global arrays
-----------------------------------
  No active global arrays


MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        28	        62
	current total bytes		         0	         0
	maximum total bytes		  20744656	  30269048
	maximum total K-bytes		     20745	     30270
	maximum total M-bytes		        21	        31


                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

          E. Apra, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski,
       T. P. Straatsma, M. Valiev, H. J. J. van Dam, Y. Alexeev, J. Anchell,
       V. Anisimov, F. W. Aquino, R. Atta-Fynn, J. Autschbach, N. P. Bauman,
     J. C. Becca, D. E. Bernholdt, K. Bhaskaran-Nair, S. Bogatko, P. Borowski,
         J. Boschen, J. Brabec, A. Bruner, E. Cauet, Y. Chen, G. N. Chuev,
      C. J. Cramer, J. Daily, M. J. O. Deegan, T. H. Dunning Jr., M. Dupuis,
   K. G. Dyall, G. I. Fann, S. A. Fischer, A. Fonari, H. Fruchtl, L. Gagliardi,
      J. Garza, N. Gawande, S. Ghosh, K. Glaesemann, A. W. Gotz, J. Hammond,
       V. Helms, E. D. Hermes, K. Hirao, S. Hirata, M. Jacquelin, L. Jensen,
   B. G. Johnson, H. Jonsson, R. A. Kendall, M. Klemm, R. Kobayashi, V. Konkov,
      S. Krishnamoorthy, M. Krishnan, Z. Lin, R. D. Lins, R. J. Littlefield,
      A. J. Logsdail, K. Lopata, W. Ma, A. V. Marenich, J. Martin del Campo,
   D. Mejia-Rodriguez, J. E. Moore, J. M. Mullin, T. Nakajima, D. R. Nascimento,
    J. A. Nichols, P. J. Nichols, J. Nieplocha, A. Otero-de-la-Roza, B. Palmer,
    A. Panyala, T. Pirojsirikul, B. Peng, R. Peverati, J. Pittner, L. Pollack,
   R. M. Richard, P. Sadayappan, G. C. Schatz, W. A. Shelton, D. W. Silverstein,
   D. M. A. Smith, T. A. Soares, D. Song, M. Swart, H. L. Taylor, G. S. Thomas,
            V. Tipparaju, D. G. Truhlar, K. Tsemekhman, T. Van Voorhis,
      A. Vazquez-Mayagoitia, P. Verma, O. Villa, A. Vishnu, K. D. Vogiatzis,
        D. Wang, J. H. Weare, M. J. Williamson, T. L. Windus, K. Wolinski,
        A. T. Wong, Q. Wu, C. Yang, Q. Yu, M. Zacharias, Z. Zhang, Y. Zhao,
                                and R. J. Harrison
                        "NWChem: Past, present, and future
                         J. Chem. Phys. 152, 184102 (2020)
                               doi:10.1063/5.0004997

                                      AUTHORS
                                      -------
        E. Apra, E. J. Bylaska, N. Govind, K. Kowalski, D. Mejia-Rodriguez,
        N. P. Bauman, A. Panyala, R. J. Harrison, M. Valiev, W. A. de Jong,
       T. P. Straatsma, H. J. J. van Dam, D. Wang, T. L. Windus, J. Hammond,
     J. Autschbach, R. de P. Soares, A. Kunitsa, K. Bhaskaran-Nair, J. Brabec,
      K. Lopata, S. A. Fischer, S. Krishnamoorthy, M. Jacquelin, R. Goswami,
       A. Woods, W. Ma, M. Klemm, O. Villa, Y. Chen, V. Anisimov, F. Aquino,
     S. Hirata, M. T. Hackler, E. Hermes, L. Jensen, J. E. Moore, J. C. Becca,
      V. Konjkov, T. Risthaus, M. Malagoli, A. Marenich, A. Otero-de-la-Roza,
        J. Mullin, P. Nichols, R. Peverati, J. Pittner, Y. Zhao, P.-D. Fan,
        A. Fonari, M. J. Williamson, J. R. Rehr, M. Dupuis, D. Silverstein,
    D. M. A. Smith, J. Nieplocha, V. Tipparaju, M. Krishnan, B. E. Van Kuiken,
        A. Vazquez-Mayagoitia, M. Swart, Q. Wu, T. Van Voorhis, A. A. Auer,
     M. Nooijen, L. D. Crosby, E. Brown, G. Cisneros, G. I. Fann, H. Fruchtl,
   J. Garza, K. Hirao, R. A. Kendall, J. A. Nichols, K. Tsemekhman, K. Wolinski,
     J. Anchell, D. E. Bernholdt, P. Borowski, T. Clark, D. Clerc, H. Dachsel,
   M. J. O. Deegan, K. Dyall, D. Elwood, E. Glendening, M. Gutowski, A. C. Hess,
         J. Jaffe, B. G. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin,
   R. Littlefield, X. Long, B. Meng, T. Nakajima, S. Niu, L. Pollack, M. Rosing,
   K. Glaesemann, G. Sandrone, M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe,
                               A. T. Wong, Z. Zhang.

 Total times  cpu:        2.9s     wall:        2.9s
